### DISCUSSION TITLE
Inbound integration type

### DISCUSSION THREAD

**Discussion by zdunecki** [saleor/apps#982] [Q&A] [answered]
Hey,

Do you have plans to implement inbound integration possibilities where Saleor can exist on third-party marketplaces? For example, let's say Zapier would use your API on behalf of users. The user will be authorized behind OAuth between the Saleor and the third-party app (Zapier in our example use case).

<img width="1793" alt="Screenshot 2023-09-05 at 14 20 25" src="https://github.com/saleor/apps/assets/18244904/30d7aebc-1eb3-4fb6-9c98-42ce52435d90">

The above example shows what it could look like in Zapier. After clicking `Sign in` button user is authorized to the Saleor OAuth consent screen.

Best,

**Comment by lkostrowski** [comment, marked as answer]
Hey, we are definitely open for official integrations on the foreign marketplaces. However we don't have any specific dates on that.

I know it will be best from the DX perspective to have it running out of the box. At the moment, you can connect Saleor with platforms like zapier via webhooks (single direction, Saleor -> Zapier) or more complex apps (calling Saleor API with app tokens)
